### PR TITLE
Feature/delete address

### PR DESCRIPTION
**Pull request by Rickytony**

### CODE REVIEW

File: app/controllers/sprangular/addresses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Sprangular::AddressesController < Sprangular::BaseController
+  before_filter :check_authorization
+
+  def destroy
+    authorize! :update, @user
+
+    address = @user.addresses.find params[:id]
+    address.destroy
+
+    respond_with address
```
Comment (joshnuss) [inline app/controllers/sprangular/addresses_controller.rb]: @Rickytony we can use the serializer here, like this:

```
render json: address,
           scope: current_spree_user,
           serializer: Sprangular::AddressSerializer,
           root: false
```

**Comment by joshnuss** [comment]
@Rickytony Looks excellent! (added one comment)

**Comment by bryanmtl** [comment]
@Rickytony When you have a chance, can you add the serializer as @joshnuss pointed out?

**Comment by Rickytony** [comment]
@bryanmtl doing it now.

**Comment by bryanmtl** [comment]
This is looking good, thanks @Rickytony.

**Pull request merged**